src/demo-app/index.html

```diff
@@ -5,7 +5,11 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>Material2</title>
   <base href="/">
-  {{content-for 'head'}}
+
+  {{#unless environment.production}}
+  <script src="/ember-cli-live-reload.js" type="text/javascript"></script>
+  {{/unless}}
```

Comment:
Can you add comments to this templating so that passerby who look at our index.html know what it is?